dfhack.init-example

```diff
@@ -223,7 +228,7 @@ enable \
 # You can comment out the extension of a line.
 
 # enable mouse controls and sand indicator in embark screen
-embark-tools enable sand mouse
+embark-tools enable sticky sand mouse
```

Comment:
This one's really a personal preference, in my opinion. It also has issues with selecting uninhabitable sites, which vanilla DF avoids.